doc/rfd/0000/README.md

```diff
@@ -0,0 +1,111 @@
+promoting it to ```discussion```.
+
+Documents under active discussion within the team must be in ```discussion``` state. When reaching
+this state the scope of the RFD must be clear and well defined.
+
+Once the discussion converges to clear understanding of what needs to be done, then the state must
+be set to ```published```.
+
+Any comment in the ```published``` state must be raised as issues or if the comment brings
+fundamental changes to the original content of the RFD then a new RFD must be created.
+
+The ```abandoned``` state indicates that the RFD will not be implemented and it can be switched from
+any previous state. Depending of the current RFD state, it may be the author(s) making an individual
+decision (```draft```) or a collective decision (```discussion``` or ```published```).
+
+## Workflow
+
+Some of these steps may have some script support in the future to make everyones life easier.
+
+### Find Next RFD Number
+
+Check for the next RFD number by going through the RFDs table.
+
+### Create a Branch & Placeholder
+
+Create a RFD branch using the name after the last (i.e. 0009) known RFD number:
+
+``` shell
+# git checkout -b rfd-0010
+```
+
+and create the placeholder:
+
+``` shell
+# mkdir -p docs/rfd/0010
+# touch docs/rfd/0010/README.md
+```
+
+Fill in the RFD authors and set the state to ```draft```.
+
+### Push Branch Remotely
+
+Push the changes to your RFD branch.
+
+``` shell
+# git add docs/rfd/0010/README.md
+# git commit -m "rfd-0010: adding placeholder for RFD..."
+# git push origin 0010
+```
+
+In the future, after the branch pushed, the table in the README will be automatically updated to
+reflect the new RFD or the change of state as well.
+
+Now is time for you to fill the RFD with your thoughts to the point where you think it's good to
+start sharing with others, to get their feedback and have a productive discussions.
+
+### Open a Pull Request for Discussion
+
+When ready to share with the team, then change the state from ```draft``` to ```discussion```
+and push it remotely.
+
+```shell
+# git commit -m "rfd-0010: change RFD state to discussion"
+# git push origin rfd-0010
+```
+
+Do not forget to open a pull request (maybe later we will have a bot for this) with the title
+_RDF:num_ .
+
+With the pull request open anyone subscribed to the repository will be notified about the new RFD
+and it can givethe feedback through pull request comments.
+
+It's up to the author(s) to incorporate any pull request comment into the RFD.
+
+### Merge the Pull Request
+
+After a while, and with team members feedback, the author(s) would merge the pull request into the
+master branch, but before that it must the state to ```published```. At this moment, the RFD must be
+ready for implementation.
+
+If an RFD state is switched to ```abandoned``` the pull request must be merged into the master
+branch. If the RFD is in a ```draft``` state then a pull request must be open and merged for future reference.
```

Comment:
```suggestion
branch. If the RFD is in a ```draft``` state then a pull request must be opened and merged for future reference.
```

Practically speaking, this means we still need to involve somebody else to abandon a RFD?  I wonder if it would make more sense for RFDs that never reach the `discussion` stage to be abandoned without merging.